README.md

```diff
@@ -6,21 +6,24 @@ Since this is such a broad and somewhat ambiguous process, there is no single co
 Thus, it's advantageous to get as much developer feedback as possible to ensure that the updated structure is easy to change, compartmentalize, test etc.
```

Comment:
All this stuff added for the "refactor" should be move to the description of an issue rather that modifying the README.md of the project :)